paddle/fluid/operators/math/blas.h

```diff
@@ -176,11 +177,24 @@ class Blas {
                    int K, T alpha, const T* A, const T* B, T beta, T* C,
                    int batchCount, int64_t strideA, int64_t strideB) const;
 
+  template <typename T>
+  void BatchedGEMM(CBLAS_TRANSPOSE transA, CBLAS_TRANSPOSE transB, int M, int N,
+                   int K, T alpha, std::vector<const T*>* a_array,
```

Comment:
`const std:vector<const T*>* ` ?

Reply:
Now Bingyang are ready to re-implement the pass in future. This PR will be aborted.